charts/fluent-bit/values.yaml

```diff
@@ -70,12 +70,11 @@ env: []
 
 envFrom: []
 
-extraServices: []
-# extraServices:
-  # - port: 1234
-  #   containerPort: 5678
-  #   protocol: TCP
-  #   name: http2
+extraPorts: []
+#   - port: 1234
+#     containerPort: 5678
```

Comment:
```suggestion
#   - port: 5170
#     containerPort: 5170
```

nit: forgive my ocd but let's use the port listed in https://docs.fluentbit.io/manual/pipeline/inputs/tcp in the commented out example port